crates/pop-common/src/errors.rs

```diff
@@ -13,12 +13,19 @@ pub enum Error {
 	Git(String),
 	#[error("IO error: {0}")]
 	IO(#[from] std::io::Error),
-	#[error("Failed to get manifest path: {0}")]
-	ManifestPath(String),
+	/// An error occurred while attempting to create a keypair from the provided URI.
```

Comment:
If docs are added it would be good to add for the Error type and other fields as well :)

Reply:
Added comments for the Error type and all the errors introduced in this PR in this commit https://github.com/r0gue-io/pop-cli/pull/316/commits/7d7df02deee57a437203a1de8f2dbfe52b38b4d7. 

For existing methods, I feel this should be addressed in a separate PR, as this one is already a big one and adding unrelated changes would make it more complex to review.

Reply:
Sounds good, would be good to do though separately to master